services/release/release.go

```diff
@@ -57,9 +57,16 @@ func createTag(gitRepo *git.Repository, rel *models.Release) error {
 		if err != nil {
 			return fmt.Errorf("CommitsCount: %v", err)
 		}
+
+		u, err := models.GetUserByEmail(commit.Author.Email)
```

Comment:
Should we try another emails of the user?

Reply:
I don't think so. This is the only email address we have access to at this point

Reply:
The tagger may not be the commit author  or commit committer.